File: exercicios/python/Exercicio05/administrativo.py
from funcionario import Funcionario
class Administrativo(Funcionario):
    def __init__(self,departamento: str, cpf: int):
        super().__init__(departamento, cpf, 10)
    
    def emprestar (self, titulo_do_livro:str):
        return "Funcionario administrativo " + super().emprestar(titulo_do_livro)

    def devolver (self, titulo_do_livro:str):
        return "Funcionario administrativo " + super().devolver(titulo_do_livro)

File: exercicios/java/Exercicio05/src/main/java/org/example/Administrativo.java
package org.example;

public class Administrativo extends Funcionario {
    public Administrativo(String departamento, long cpf, int diasDeEmprestimo) {
        super(departamento,cpf, diasDeEmprestimo);
    }

    @Override
    public String devolver(String tituloLivro) {
        String devolucaoDoUsuario = super.devolver(tituloLivro);
        return "Administrativo " + devolucaoDoUsuario;
    }
    @Override
    public String emprestar(String tituloLivro) {
        String emprestarDoUsuario = super.emprestar(tituloLivro);
        return "Administrativo " + emprestarDoUsuario;
    }
}
